src/components/Comments.jsx

```diff
@@ -26,6 +29,27 @@ export function Comments({setCommentCount, commentCount}) {
         return <p>No comments yet...</p>
     }
 
+    function handleDelete(commentToRemove) {
+       deleteComments(commentToRemove.comment_id).then(() => {
+        setAlertMessage('Comment Deleted!')
+       })
+       .catch((err)=> {
+        setAlertMessage('Error deleting comment, try again later')
+       })
+       setComments((currComments) => {
+        const filteredComments = currComments.filter((comment) => {
+            return comment.comment_id !== commentToRemove.comment_id
+        })
+        return filteredComments
+        })
+        const updatedCommentCount = commentCount - 1
+        setCommentCount(updatedCommentCount)
+        setShowAlertMessage(true)
+        setTimeout(() => {
+            setShowAlertMessage(false)
+        }, 4000)
```

Comment:
The alert message should show in the .then or catch block. You shouldn't need a setTimeout for this.